app/controllers/admin/consultations_controller.rb

```diff
@@ -51,6 +65,13 @@ def unfeatured
 		redirect_back fallback_location: root_path,  flash_success_info: 'Consultation was successfully updated as unfeatured.'
 	end
 
+	def check_active_ministry
```

Comment:
@BALAJI-BALA - what exactly does this block do bro? Sorry I'm not too clear from the code, we can also meet tomorrow and discuss this!